schema/vrs.yaml

```diff
@@ -187,7 +187,34 @@ definitions:
           - $ref: "#/definitions/Location"
           - $ref: "#/definitions/CURIE"
       copies:
-        $ref: "#/definitions/AbsoluteCopyCount"
+        oneOf:
```

Comment:
This will be easier to use if it's always a Range, possibly with equal endpoints.